packages/web-app-admin-settings/src/components/Spaces/BatchActions.vue

```diff
@@ -0,0 +1,77 @@
+<template>
```

Comment:
What's the reason for implementing the BatchActions component so very specific in the `Spaces` components section? I think this is a perfect example for a generic component which can be added to the `AppTemplate`. 🤔

Reply:
Hmm yeah, good point. Would remove boilerplate for https://github.com/owncloud/web/pull/8317 as well. We just need to pass the mixin-actions to the BatchActions component then.

Reply:
>  We just need to pass the mixin-actions to the BatchActions component then

Can we do that though? I guess we register the mixins in the parent component and then pass the methods to `BatchActions`.

Reply:
@kulmann I've done that in https://github.com/owncloud/web/pull/8317, as that PR will get merged sooner probably.